youths/signals.py

```diff
@@ -0,0 +1,21 @@
+from django.conf import settings
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from youths.models import YouthProfile
+
+
+@receiver(post_save, sender=YouthProfile)
+def generate_membership_number(sender, instance, created, **kwargs):
+    """Generate a membership number for a youth profile.
+
+    Membership number is only generated for a new profile.
+    """
+    if created:
```

Comment:
Is it a good restriction (right now) to only generate the number for new instances? Also I wonder if a data migration would be needed for existing data?

Reply:
I think generating this only for new instances is a good restriction. Data migration would be nice 👍